builtin/logical/transit/path_encrypt.go

```diff
@@ -57,12 +73,18 @@ func pathEncryptWrite(
 
 	// Error if invalid policy
 	if p == nil {
-		p, err = generatePolicy(req.Storage, name)
+		p, err = generatePolicy(req.Storage, name, len(context) != 0)
```

Comment:
This seems a little weird, since there's not a lot of context about what this third parameter will be. Should we do something like 

```go
isDerived := len(context) != 0
generatePolicy(req.Storage, name, isDerived)
```